### PR TITLE
fix: integration miss related to camera lidar fusion

### PR DESCRIPTION
**Pull request by yukkysaito**
Signed-off-by: Yukihiro Saito <yukky.saito@gmail.com>

## Description(required)
Fixed a porting error in camera lidar fusion when porting to autoware.auto.

## Review Procedure(required)
Confirmed to work with tier4 rosbag on my environment.

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Read [commit-guidelines][commit-guidelines]
- [x] Assign PR to reviewer

If you are adding new package following items are required:

- [ ] Documentation with description of the package is available
- [ ] A sample launch file and parameter file are available if the package contains executable nodes

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [x] Commits are properly organized and messages are according to the guideline
- [x] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[commit-guidelines]: https://www.conventionalcommits.org/en/v1.0.0/
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml [hunk 1]
```diff
@@ -86,25 +85,48 @@
...
 
+  <!-- Merger -->
   <include file="$(find-pkg-share object_merger)/launch/object_association_merger.launch.xml">
     <arg name="input/object0" value="$(var lidar_detection_model)/objects"/>
     <arg name="input/object1" value="euclidean/camera_lidar_fusion/short_range_objects"/>
+    <arg name="output/object" value="camera_lidar_fusion/objects"/>
+  </include>
+  <include file="$(find-pkg-share object_merger)/launch/object_association_merger.launch.xml">
+    <arg name="input/object0" value="detection_by_tracker/objects"/>
+    <arg name="input/object1" value="camera_lidar_fusion/short_range_objects"/>
```
[the remark this answers is not in the document]
Comment (yukkysaito) [reply launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml:128]: @miursh Is the rosbag data correct? 
It is working fine in my environment.
Please check ros2 topic and ros2 node information. :pray:

[the remark this answers is not in the document]
Comment (yukkysaito) [reply launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml:128]:
```suggestion
    <arg name="input/object1" value="camera_lidar_fusion/objects"/>
```

[the remark this answers is not in the document]
Comment (yukkysaito) [reply launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml]: [ee61737](https://github.com/autowarefoundation/autoware.universe/pull/481/commits/ee6173708fcfde4f3a15b73d5f5741ffbfa29f78)

**Review by miursh — APPROVED** [review]
LGTM

**Pull request merged**